package.json

```diff
@@ -51,7 +54,9 @@
   },
   "scripts": {
     "test": "mocha",
-    "compile": "babel src --out-dir dist",
+    "compile:cjs": "babel src --out-dir dist",
+    "compile:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "compile": "npm-run-all --parallel compile:cjs compile:es",
```

Comment:
I'm unsure how long it takes to compile either package or how big `npm-run-all` is but maybe you can explain why we should use that over: `npm run compile:cjs & npm run compile:es`.

(also i think it could be shortened to `run-p compile:*` but i don't know if we should...)